style.css

```diff
@@ -141,3 +141,78 @@ body {
 .dbg {
   border: 2px solid red;
 }
+
+@media screen and (max-width: 900px) {
+  .game-container {
+    flex-direction: column;
+  }
+
+  .quiz-question {
+    position: absolute;
```

Comment:
I don't think we need position absolute for making the page responsive. It can be done with flexbox alone I think. Let me know if there was some issue with it.

Reply:
The problem with position absolute is that it might look good on your phone but it will not look fine on other person's device with different dimensions.

Reply:
I used the absolute position to avoid the displacement of the counter depending on the size of the question, but if it doesn't render well, I'll leave that as a simple flexbox :) (and sorry, I don't know about Road To Hacktober program, this is my first time with Hacktober Fest)

Reply:
not an issue, hope you have a great Hacktoberfest!